.editorconfig

```diff
@@ -44,10 +44,10 @@ dotnet_diagnostic.xUnit2018.severity = suppress # "do not compare an object's ex
 # Sort using and Import directives with System.* appearing first
 dotnet_sort_system_directives_first = true
 # Avoid "this." and "Me." if not necessary
-dotnet_style_qualification_for_field = false:suggestion
-dotnet_style_qualification_for_property = false:suggestion
-dotnet_style_qualification_for_method = false:suggestion
-dotnet_style_qualification_for_event = false:suggestion
+dotnet_style_qualification_for_field = false:none
+dotnet_style_qualification_for_property = false:none
+dotnet_style_qualification_for_method = false:none
+dotnet_style_qualification_for_event = false:none
```

Comment:
❗️  These should be `silent` (or `refactoring`) so the fading and Fix All still works.

Reply:
Done.